src/index.js

```diff
@@ -1,7 +1,9 @@
 const {Client} = require('./client');
 const {consume} = require('./consumer');
+const {credentials} = require('./credentials');
 
 module.exports = {
   Client,
   consume,
+  credentials,
```

Comment:
As I understand it, it would look like

```javascript
const {Client, credentials} = require('taskcluster-lib-pulse');

...

new Client({
  credentials: credentials.claimedCredentials({ ..}),
  ...
});
```

that "stutters" a little -- the word "credentials" appears several times.  I think it would be better to export the individual credential functions at the top level of the module, so you could write

```javascript
const {Client, claimedCredentials} = require('taskcluster-lib-pulse');
```